helm/hpcc/templates/_helpers.tpl

```diff
@@ -991,6 +993,12 @@ spec:
   selector:
     server: {{ .selector | quote }}
   type: {{ $lvars.type }}
+{{- if $lvars.loadBalancerSourceRanges }}
```

Comment:
should this also be dependent on $lvars.type ? i.e. don't generate if ClusterIP type..

Reply:
Or perhaps give an error. I did wonder, but was not sure whether it's worth the bother.

Reply:
not sure, I assume it's benign if generated on a ClusterIP service, though it still implies a misconfiguration of sorts.
I think an error is preferable to ignoring.

Reply:
Error check added